index.html

```diff
@@ -69,7 +76,7 @@ <h1>I'm Naan. <br>Glad to see <br>you!</h1>
                 <button class="see_project">See project</button>
```

Comment:
- [ ] consider increasing the width of this button in order to make it look like the one in Figma remember you are requested to stick to the provided design as much as possible and this will help you to fulfill that requirement

Reply:
You just added a new requirement here. I dont get this.